docs/src/test/java/sample/javadsl/ConsumerExample.java

```diff
@@ -388,4 +388,20 @@ public void demo() {
   }
 }
 
+class TransactionalSource extends ConsumerExample {
+  public static void main(String[] args) {
+    new TransactionalSource().demo();
+  }
 
+  public void demo() {
+    // #transactionalSource
+    Consumer
+      .transactionalSource(consumerSettings, Subscriptions.topics("source-topic"))
+      .via(business())
+      .map(msg ->
+        new ProducerMessage.Message<byte[], String, ConsumerMessage.PartitionOffset>(
+          new ProducerRecord<>("sink-topic", msg.record().value()), msg.partitionOffset()))
+      .runWith(Sink.ignore(), materializer);
```

Comment:
Does it make sense to show a `transactionalSource` without a `transactionalSink`? Is this snippet even referenced?

Reply:
You're right, this wasn't used and wouldn't make much sense anyway.  It was a remnant from an earlier documentation attempt (documenting it across existing consumer and producer doc pages) that I later abandoned.